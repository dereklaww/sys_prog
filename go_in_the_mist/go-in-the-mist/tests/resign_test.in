who
place A1
place B1
place A2
place B2
resign