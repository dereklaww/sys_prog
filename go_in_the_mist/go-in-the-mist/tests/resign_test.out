B
White wins!
A1B1A2B2
Thank you for playing!